telegram/message.py

```diff
@@ -808,6 +813,23 @@ def reply_poll(self, *args, **kwargs):
         self._quote(kwargs)
         return self.bot.send_poll(self.chat_id, *args, **kwargs)
 
+    def reply_dice(self, *args, **kwargs):
+        """Shortcut for::
+
+            bot.send_dice(update.message.chat_id, *args, **kwargs)
+
+        Keyword Args:
+            quote (:obj:`bool`, optional): If set to ``True``, the poll is sent as an actual reply
```

Comment:
I know where you copied this from

```suggestion
            quote (:obj:`bool`, optional): If set to ``True``, the dice is sent as an actual reply
```

Reply:
Got me :D changed.